frugalos_config/src/builder.rs

```diff
@@ -253,7 +289,7 @@ impl<'a> SegmentsBuilder<'a> {
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
 struct SlotKey {
     bucket_no: BucketNo,
-    segment_no: u16,
+    segment_no: SegmentNo,
```

Comment:
frugalos builds segment-map from scratch every starting up. I think that there is no effect with this change. 
Did you confirm it?

Reply:
I don't get what you mean, but
https://github.com/frugalos/frugalos/blob/c57344cf04d420eb3634f0767be9114fa585b4a9/frugalos_config/src/builder.rs#L13
so there is no effect. Just refactored.

Reply:
Sorry, there is no problem. It may not be necessary, I'v checked as follows.

1. Initialize by v0.12.0 binary (Put devices, buckets, objects). 
2. Stop frugalos.
3. Start frugalos by `fix/koba/evenly-distribute-segments` binary and GET previous registerd objects.